bodhi/server/services/updates.py

```diff
@@ -308,8 +309,15 @@ def query_updates(request):
         query = query.filter(Update.date_pushed < pushed_before)
 
     releases = data.get('releases')
+    current_release = data.get('current_release')
     if releases is not None:
         query = query.filter(or_(*[Update.release == r for r in releases]))
+    elif str(current_release).lower() not in ['0', 'false']:
```

Comment:
I think colander should already have turned this into a ```bool``` for you, since you added a schema for it.